### ISSUE TITLE
Machbarkeits-Check: Projektgruppe in Projektboxen anzeigen

### ISSUE THREAD

**Issue by innosmith** [decidim-swiss/decidim.swiss#25]
Gewisse Projekte sind einer Projektgruppe zugeordnet. Lässt sich der Name der Projektgruppe in der Übersicht aller Projekte zusätzlich in der Informationsbox anzeigen? Wenn positiv, kreieren wir ein Mockup zur Anpassung der Informationen.

Etwas Ähnliches haben wir bei DialogLuzern gemacht, um bei Veranstaltungen die Organisation mit anzuzeigen.

**Comment by carlobeltrame** [comment]
Ja, das müsste grundsätzlich möglich sein. Decidim hat extra Infrastruktur wo man sich drin einhängen kann und die angezeigten Felder dieser "Informationsboxen" (bei Decidim heissen sie Cells) anpassen, ergänzen oder entfernen kann. Genau so haben wir das wie du sagst bei Dialog Luzern auch schon gemacht.
